### DISCUSSION TITLE
如果要使用ChatGPT的conversation_id来保持历史对话，应该如何使用api呢？

### DISCUSSION THREAD

**Discussion by zeushera140** [linweiyuan/go-chatgpt-api#280] [Q&A]
因为readme.md只有 imitate的demo，所以我不理解怎么类似 https://github.com/acheong08/ChatGPT 这个库一样，创造conversation并且利用conversation_id或者直接在该conversation下继续对话。

感谢了，不太懂go，已有的example，我不太理解，抱歉了，https://github.com/linweiyuan/go-chatgpt-api/blob/main/example/chatgpt.http

**Comment by linweiyuan** [comment]
有例子

**Reply by zeushera140** [reply]
example文件夹里的么，我在Python里应该怎么改写对应代码呢？抱歉，确实不太懂

**Reply by zeushera140** [reply]
直接用Python对特定URL提交post需求？然后获取response？不是类似 chatcompletions那样？

**Reply by zeushera140** [reply]
搞定了……之前一直以为有Python api，谢谢了，打扰了。
